docs/edge-authorizers/deployment-and-operation.mdx

```diff
@@ -6,38 +6,33 @@ description: Deployment and Operation of edge authorizers
 
 # Deployment and Operation
 
-Edge authorizers can be deployed in various ways based on the application and target environment needs and requirements.
-In its most basic form, an authorizer is a Docker container instance, which can be deployed as a standalone service,
-sidecar, or development workstation.
+Edge Authorizers can be deployed in various ways based on the application and target environment needs and requirements.
+In its most basic form, an authorizer is a Docker container instance, which can be deployed as a standalone microservice or a sidecar.
 
-An edge authorizer always operates in the context of a (single) Aserto organization and is assigned a specific policy
+An Edge Authorizer always operates in the context of a (single) Aserto organization and is assigned a specific policy
 instance to serve. This information is provided to the edge authorizer via a configuration file, which it uses to send a
```

Comment:
Up until the 2nd sentence here we've capitalized "Edge Authorizer" and "Control plane" both are in lowers here